lib/fungus_toast/games/grid.ex

```diff
@@ -0,0 +1,32 @@
+defmodule FungusToast.Games.Grid do
+    alias FungusToast.Games.GridCell
+
+def create_starting_grid(number_of_rows, number_of_columns, player_ids) do
+    initial_grid_cells_map = map_of_player_id_to_random_grid_index(number_of_rows, number_of_columns, player_ids)
+
+    Enum.reduce(initial_grid_cells_map, %{}, fn {player_id, position}, map ->
+      Map.put(map, position, %GridCell{
+        index: position,
+        player_id: player_id,
+        live: true,
+        empty: false
+      })
+    end)
+  end
+
+  defp map_of_player_id_to_random_grid_index(number_of_rows, number_of_columns, player_ids) do
+    number_of_players = length(player_ids)
+    number_of_cells_to_choose_from = number_of_columns * number_of_rows / number_of_players
+    Enum.into(
+      0..number_of_players - 1,
+      %{},
+      &{Enum.at(player_ids, &1),
+       Enum.random(
+         trunc((&1) * (number_of_cells_to_choose_from))..trunc(
```

Comment:
if we guaranteed `number_of_cells_to_choose_from` was an integer, couldn't we get rid of the `trunc` calls in here?  I think that would just mean changing 
`number_of_cells_to_choose_from = number_of_columns * number_of_rows / number_of_players` 

to 

`number_of_cells_to_choose_from = trunc(number_of_columns * number_of_rows / number_of_players)`